files/jvm/src/main/native/Makefile

```diff
@@ -58,6 +70,16 @@ $(BUILD_DIR)/i686/linux/$(QUICKLIST_SOURCE).o: posix/$(QUICKLIST_SOURCE).cc
 	mkdir -p $(BUILD_DIR)/i686/linux; \
 	$(CC) -c -m32 $< $(CCFLAGS) $(JNI_INCLUDE) -fPIC -o $@
 
+$(BUILD_DIR)/x86_64/freebsd/$(QUICKLIST_SOURCE).o: posix/$(QUICKLIST_SOURCE).cc
+	echo $(JNI_INCLUDE)
+	mkdir -p $(BUILD_DIR)/x86_64/freebsd; \
+	$(CC) -c $< $(CCFLAGS) $(JNI_INCLUDE) -fPIC -o $@
+
+$(BUILD_DIR)/i686/freebsd/$(QUICKLIST_SOURCE).o: posix/$(QUICKLIST_SOURCE).cc
+	echo $(JNI_INCLUDE)
```

Comment:
I don't really care, but I'm not sure if it was intentional to leave in these echos.

Reply:
Well, it was a copy&paste from the linux targets. I suppose they were added because the logic to find JNI_INCLUDE is not rock solid, I'll replace it with the use of JAVA_HOME.

Reply:
Right. I knew that looked familiar.